.gitignore

```diff
@@ -7,3 +7,6 @@ doc/build
 
 # Ignore macOS cache files
 .DS_Store
+
+# Ignore testsuite file/directories
```

Comment:
I guess while we are at it we could also ignore `caselist.*` files generated by cice.setup

Reply:
good idea, i'll add that in both cice and icepack and update shortly.